fastlane_core/spec/cert_checker_spec.rb

```diff
@@ -55,7 +55,6 @@ class ProcessStatusMock
     describe '#install_missing_wwdr_certificates' do
       it 'should install all official WWDR certificates' do
         allow(FastlaneCore::CertChecker).to receive(:installed_wwdr_certificates).and_return([])
-        expect(FastlaneCore::CertChecker).to receive(:install_wwdr_certificate).with('G1')
```

Comment:
Does this deletion (and the others like it later in the PR) assert that the G1 is **absent** from the set of installed certs? OR does it merely stop caring about the G1?

We want the first.

*NOTE: I'm have little experience with Ruby development, so my concern here may be unfounded.*